lib/msal-browser/README.md

```diff
@@ -86,18 +86,19 @@ If you have MSAL v1.x currently running in your application, you can follow the
 
 ## Samples
 
-The [`VanillaJSTestApp2.0` folder](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples) contains sample applications for our libraries. You can run any sample by changing the `authConfig.js` file in the respective folder to match your app registration and running the `npm` command `npm start -- -s <sample-name> -p <port>`.
+The [`msal-browser-samples` folder](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples) contains sample applications for our libraries. You can run any VanillaJSTestApp2.0 sample by changing the `authConfig.js` file in the respective folder to match your app registration and running the `npm` command `npm start -- -s <sample-name> -p <port>` in the VanillaJSTestApp2.0 folder. The TypescriptTestApp2.0 sample can be run with `npm start` in its folder.
```

Comment:
Does it makes sense to itemize the points mentioned here regarding samples. Minor nitpick though.

Reply:
Done.